var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -143,6 +147,8 @@ def configure(self):
         if self.spec.satisfies('platform=darwin'):
             cflags.append('-DCOMMON_PTHREAD_BARRIER')
             ldflags.append('-lm -pthread')
+        elif self.spec.satisfies('platform=bgq'):
+            ldflags.append('-lm -lrt -lpthread')
```

Comment:
Isn't this redundant?  These flags are already added if the platforms isn't `darwin`...